libraries/shared/factories/storage/transformer.go

```diff
@@ -20,10 +20,18 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/makerdao/vulcanizedb/libraries/shared/storage"
 	"github.com/makerdao/vulcanizedb/libraries/shared/storage/types"
-	"github.com/makerdao/vulcanizedb/libraries/shared/transformer"
 	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
 )
 
+type ITransformer interface {
+	Execute(diff types.PersistedDiff) error
+	KeccakContractAddress() common.Hash
```

Comment:
Really minor and not a blocker to this PR but would it make sense to memoize the result for the implementation of this function on ln 56? Thinking we could have a private var on the Transformer struct and only compute the hash at runtime if it hasn't already been called and assigned